### PR TITLE
Reduce logging branch path size.

### PR DESCRIPTION
**Pull request by jkotalik**
We were hitting max path issues.

There were also two target frameworks, so removing one as cleanup.

Fixes https://github.com/aspnet/AspNetCore-Internal/issues/2665

### CODE REVIEW

**Review by dougbu — APPROVED** [review]
😺

**Comment by jkotalik** [comment]
@aspnet-hello triage https://github.com/aspnet/AspNetCore/pull/11297/checks?check_run_id=150199927

File: build/SharedFx.targets [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <FxProjectToBuild Condition=" '$(TargetRuntimeIdentifier)' == 'linux-x64' AND '$(LinuxInstallerType)' == 'rpm' " Include="$(RepoRoot)src\Installers\Rpm\**\*.*proj" />
 
     <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepoRoot)src\SiteExtensions\Runtime\Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj" />
-    <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepoRoot)src\SiteExtensions\LoggingBranch\LoggingBranch.csproj" />
+    <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepoRoot)src\SiteExtensions\LoggingBranch\LB.csproj" />
```
Comment (davidfowl) [inline build/SharedFx.targets]: 😆

**Comment by JunTaoLuo** [comment]
This should help with the build hangs too.

**Review by JunTaoLuo — APPROVED** [review]

**Comment by jkotalik** [comment]
@aspnet/build please merge this if we have another flaky test. I'm making a PR now to mark it as flaky.

**Comment by dougbu** [comment]
@jkotalik sure, but let's let the Ubuntu test run complete before we do that. (Seemed to take a while to be queued but has been running fine for a half hour.) I'll check back before I leave (soon) and, if necessary, after I get home.

**Pull request merged**